Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -1425,6 +1426,25 @@ def get_current_time_in_seconds() -> float:
     return time.time_ns() / (10 ** 9)
 
 
+def create_files_to_upload(file_mime_type: str, file_name: str, file_bytes: bytes, attachment_name: str | None = None) -> dict:
+    """ Creates the file object to upload to Jira
+        Args:
+            file_mime_type (str): The mime type of the file.
+            file_name (str): The name of the file.
+            file_bytes(bytes): The bytes of the file.
```

Comment:
Do the bytes of the file mean the content?

Reply:
yes